stripe/src/test/java/com/stripe/android/paymentsheet/PaymentSheetActivityTest.kt

```diff
@@ -29,17 +38,36 @@ class PaymentSheetActivityTest {
 
     private val testCoroutineDispatcher = TestCoroutineDispatcher()
 
-    private val viewModel = PaymentSheetViewModel(
-        application = ApplicationProvider.getApplicationContext(),
-        publishableKey = ApiKeyFixtures.FAKE_PUBLISHABLE_KEY,
-        stripeAccountId = null,
-        stripeRepository = FakeStripeRepository(),
-        workContext = testCoroutineDispatcher
+    private val paymentIntent: PaymentIntent = mock()
```

Comment:
Can we use something from `PaymentIntentFixtures` instead of a mock?

Reply:
oops this still needs to be deleted